script/rsync.sh

```diff
@@ -79,6 +79,12 @@ set +e
     [ ! -L "$subfolder/.run_last" ] || rm "$subfolder/.run_last"
     ln -s -T "$(pwd)/$logdir" $subfolder/.run_last
 
+    # run additional hook script from current subfolder
+    [[ -x "$subfolder/hook_script" ]] && "$subfolder/hook_script" 2>$logdir/hook_script.err > $logdir/hook_script.out
+
+    # run additional hook script from script folder
+    [[ -x "$DIR/generic_hook" ]] && "$DIR/generic_hook" "$environ" 2>$logdir/generic_hook.err > $logdir/generic_hook.out
+
```

Comment:
this looks like it would be what we need but to have that mergeable the example script above shouldn't be commited directly. Instead you could put it into a documentation in the README of this project.

Reply:
Yes, this is just a draft to show what it would look like for real.

Reply:
I would consider 106 lines way too much for an inline code snippet.

Reply:
@kalikiana not script/rsync.sh which is not a hook script but script/generic_hook which is 9 lines